js_modules/dagster-ui/packages/ui-core/src/runs/RunTableEmptyState.tsx

```diff
@@ -0,0 +1,38 @@
+import {Box, Icon, NonIdealState} from '@dagster-io/ui-components';
+
+import {AnchorButton} from '../ui/AnchorButton';
+
+export const RunTableEmptyState = ({anyFilter}: {anyFilter: boolean}) => {
```

Comment:
`isFiltered` ?

Reply:
Will change this in the follow-up with the rest of the grouped view updates!